components/permit-holders/DoctorInformationCard.tsx

```diff
@@ -56,29 +60,31 @@ export default function DoctorInformationCard(props: DoctorInformationProps) {
         </Box>
         <Box>
           <Text as="p" textStyle="body-regular">
-            {physician.addressLine1}
+            {physician?.addressLine1 || ''}
           </Text>
           <Text as="p" textStyle="body-regular">
-            {physician.addressLine2}
+            {physician?.addressLine2 || ''}
           </Text>
           <Text as="p" textStyle="body-regular">
-            {`${physician.city} ${physician.province}`}
+            {`${physician?.city || ''} ${physician?.province || ''}`}
           </Text>
           <Text as="p" textStyle="body-regular">
             {`Canada`}
           </Text>
           <Text as="p" textStyle="body-regular">
-            {physician.postalCode}
+            {physician?.postalCode || ''}
           </Text>
         </Box>
       </VStack>
 
       <Flex w="100%" justifyContent="flex-end" paddingTop="8px">
-        <PreviousDoctorsInformationModal>
-          <Button color="primary" variant="ghost" textDecoration="underline">
-            <Text textStyle="body-bold">{'View previous doctors'}</Text>
-          </Button>
-        </PreviousDoctorsInformationModal>
+        {props.previousPhysicianData && (
```

Comment:
If we make `previousPhysicianData` non-optional, this condition should change to `previousPhysicianData.length > 0`